client/src/app/components/menu/menu-item.ts

```diff
@@ -11,16 +11,16 @@ export class MenuItem {
     public readonly children: MenuItem[];
 
     constructor(name: string,
-                streams?: MenuItemUpdates = {},
-                children?: MenuItem[] = []) {
```

Comment:
Is there any particular reason for removing defaults and setting values with `|| []`?